### PR TITLE
add v2.3.0 post

### PR DESCRIPTION
**Pull request by krancour**
Signed-off-by: Kent Rancourt <kent.rancourt@microsoft.com>

### CODE REVIEW

File: content/2022-02-10-brigade-2.3.0.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+"Brigade in Five Minutes" series focused on authentication using third-party
+identity providers, and because we anticipate the next episode covering
+management of user permissions, now seems like an appropriate time to turn the
+spotlight on a specific new feature that streamlines that process.
+
+Beginning with Brigade v2.3.0, if you leverage a third-party identity provider
+like GitHub or an Azure Active Directory tenant to authenticate your users,
+there is now an option to _automatically_ grant users global, read-only
+permissions the first time they federate. This can be a tremendous convenience
+for Brigade administrators who already have a good handle on _who_ can
```
Comment (karenhchu) [inline content/2022-02-10-brigade-2.3.0.md:26]:
```suggestion
for Brigade administrators that already have a good handle on _who_ can
```

Comment (karenhchu) [reply content/2022-02-10-brigade-2.3.0.md:26]: Not sure if this is the best suggestion but it feels weird to have two "who"s so close to each other.

Comment (krancour) [reply content/2022-02-10-brigade-2.3.0.md]: Fixed in #e77921ec5ed516e962496474ae6f842126506a72

**Comment by krancour** [comment]
Although @karenhchu has approved this, not one's going to notice a blog post late on Firday, so I changed the date to 2/14 and will hold it for Monday.

This means more feedback is welcome in the meantime.

**Pull request merged**